### PR TITLE
Updated QueryParameters dictionary in HttpRequest.cs to use an object…

### PR DESCRIPTION
**Pull request by lucasbode**
It looks like in the recent release candidate releases the HttpPredicateFields was objected to allow objects in for QueryParameters property. This is useful because I've sometimes run across the issue of api's using a key multiple times in the query string. Ex api/v1/customer?id=123&id=456&id=789.

However when retrieving those requests from the imposter an error is thrown because the HttpRequest object can only handle the string for it's dictionary value. This pull request resolves that issue.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,3 +8,4 @@ services:
     ports:
     - "2525:2525"
     - "6000:6000"
+    - "6001:6001"
```
Comment (lucasbode) [inline docker-compose.yml:11]: I needed to expose this port in order to get the Acceptance tests to pass.

File: docker/mountebank/Dockerfile [hunk 2]
```diff
@@ -3,7 +3,7 @@ FROM mhart/alpine-node
 EXPOSE 2525
 ENTRYPOINT ["mb"]
 CMD ["start","--mock"]
-ENV MOUNTEBANK_VERSION=1.9.0
+ENV MOUNTEBANK_VERSION=2.2.0
```
Comment (lucasbode) [inline docker/mountebank/Dockerfile:6]: I also needed to update to the newest version of mountebank to get the acceptance tests to pass.

**Comment by mattherman** [comment]
Apologies for missing this case and thank you for fixing it. I'll try to get to this sometime today.

File: docker/mountebank/Dockerfile (hunk 2 above)
Comment (mattherman) [reply docker/mountebank/Dockerfile:6]: Thanks. I don't generally use docker when testing so I'm not surprised this stuff was out of date.

**Comment by mattherman** [comment]
Sorry it took me a while to get to this, looks great. Merging now and I'll push a new pre-release package with this update shortly.

**Comment by mattherman** [comment]
A new package version, 5.0.0-rc4, has been published. It should be available soon.

**Pull request merged**